services/QuillLMS/app/models/districts_admins.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: districts_admins
+#
+#  id          :bigint           not null, primary key
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  district_id :bigint           not null
+#  user_id     :bigint           not null
+#
+# Indexes
+#
+#  index_districts_admins_on_district_id  (district_id)
+#  index_districts_admins_on_user_id      (user_id)
+#
+class DistrictsAdmins < ApplicationRecord
+  belongs_to :district
+  belongs_to :user
+
+  after_create :attach_to_subscribed_schools
+  after_destroy :detach_from_schools
+
+  def admin
+    user
+  end
+
+  def attach_to_subscribed_schools
+    district = District.find(district_id)
```

Comment:
This should just be `district` per the `belongs_to` defined on line 19.  But I'm also not seeing it get used below so probably just needs to be deleted?